src/logging.rs

```diff
@@ -55,6 +55,8 @@ pub enum LogLevel {
     Default,
     // All user-facing output (+ `log::LevelFilter::Debug`).
     Verbose,
+    // Only show statistics (+ `log::LevelFilter::Info`).
+    Statistics,
```

Comment:
I think it'd be best not to tie this to log level. `--verbose` and `--quiet` should actually work with `--statistics`, I think, since some of that deals with debug logging that isn't related to that final output format.